server/parsec/components/invite.py

```diff
@@ -276,6 +281,253 @@ class InviteAsInvitedInfoBadOutcome(BadOutcomeEnum):
     INVITATION_DELETED = auto()
 
 
+# New transport definitions
+# TODO: Remove the old ones once fully migrated
+
+
+@dataclass
+class GreetingAttemptCancelledBadOutcome(BadOutcome):
+    origin: GreeterOrClaimer
+    reason: CancelledGreetingAttemptReason
+    timestamp: DateTime
+
+
+@dataclass
+class NotReady:
+    pass
```

Comment:
Here notReady doesn't need any data ?

Reply:
Yes, it's just a status. It could have been part of the `*BadOutcome` enums but it's not really a "bad" outcome, it's just a status communicating that the other peer hasn't performed this step yet.